iis-wf/iis-wf-citationmatching-direct/src/main/java/eu/dnetlib/iis/wf/citationmatching/direct/service/CitationMatchingDirectReporter.java

```diff
@@ -0,0 +1,64 @@
+package eu.dnetlib.iis.wf.citationmatching.direct.service;
+
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+
+import com.google.common.collect.Lists;
+
+import eu.dnetlib.iis.common.citations.schemas.Citation;
+import eu.dnetlib.iis.common.schemas.ReportParam;
+import pl.edu.icm.sparkutils.avro.SparkAvroSaver;
+
+/**
+ * Reporter of citation matching direct job counters.<br/>
+ * It calculates citation matching direct counters and saves them
+ * as {@link ReportParam} datastore.
+ * 
+ * @author madryk
+ */
+public class CitationMatchingDirectReporter {
+
+    private static final String MATCHED_CITATIONS_COUNTER = "export.matchedCitations.direct.total";
+    
+    private static final String DOCS_WITH_MATCHED_CITATIONS_COUNTER = "export.matchedCitations.direct.docsWithAtLeastOneMatch";
+    
+    
+    private SparkAvroSaver avroSaver = new SparkAvroSaver();
+    
+    
+    //------------------------ LOGIC --------------------------
+    
+    /**
+     * Calculates and saves citation matching counters using citations rdd
+     * under outputReportPath.
```

Comment:
I would rephrase it, currently it seems citations rdd is under `outputReportPath` ;)